src/parent/parent.js

```diff
@@ -315,7 +316,7 @@ export function parentComponent<P, X, C>({
   let currentProxyContainer: ?ProxyObject<HTMLElement>;
   let childComponent: ?ChildExportsType<P>;
   let currentChildDomain: ?string;
-  let currentContainer: HTMLElement | void;
```

Comment:
I'm a little concerned about removing the `| void` here. Wondering if we can add it back and instead add a check to make sure it's truthy before passing it to `isElementClosed()`. Ex:

```js
        const error = err || new Error(RENDER_ERRORS.COMPONENT_DESTROYED);
        if (
          (currentContainer && isElementClosed(currentContainer)) ||
          error.message === RENDER_ERRORS.NAVIGATED_AWAY
        ) {
          initPromise.resolve();
        } else {
          initPromise.asyncReject(error);
        }
```